lib/src/services/movie-service.dart

```diff
@@ -35,14 +39,32 @@ abstract class MovieService {
           MovieSearchSettings settings) =>
       _fetchPagedResult(settings, "3/movie/latest");
 
-  Future<Movie> getMovie(int id,
-      {String language = "en-US",
-      List<String> imageLanguages = const ["en", null],
-      MovieAppendSettings appendSettings,
-      QualitySettings qualitySettings}) async {
-    appendSettings = appendSettings ?? MovieAppendSettings();
-    qualitySettings = qualitySettings ?? QualitySettings();
+  Future<List<Genre>> getAllMovieGenres({String language = "en-US"}) async {
```

Comment:
I would move this to configuration service, as requests that could be cached are better suited to sit there.

Reply:
Yeah that should be fine.